.drone.yml

```diff
@@ -16,7 +16,9 @@ steps:
       from_secret: gcloud_auth
   commands:
   - docker pull --quiet rancher/hardened-build-base:v1.16.4b7
-  - dapper -f Dockerfile --target dapper make dapper-ci
+  - docker pull --quiet alpine:latest
```

Comment:
why is this necessary? Won't this be pulled on-demand for Dockerfile.windows?

Reply:
> why is this necessary? Won't this be pulled on-demand for Dockerfile.windows?

my working assumption here is that this is an attempt to more fully pre-cache images used in the build. the exact reason for the force pull of rancher/hardened-build-base escapes me but it is there for a reason

Reply:
@dweomer that's right, I wasn't sure of the reason but I wanted to mimic the same process in case there was actually an issue caused by not pre-caching the images.